packages/shared/lib/models/Connection.ts

```diff
@@ -86,4 +86,5 @@ export interface ConnectionList {
     provider: string;
     created: string;
     metadata?: Metadata | null;
+    error_log_id?: number | string | null;
```

Comment:
Number or string so that it can work with the old kids and new logs while we are in transition